EVES/eves-0001.adoc

```diff
@@ -0,0 +1,40 @@
+# 1 - EVES Purpose and Process
+
+[cols="1,5"]
+|===
+|EVES
+|1
+
+|Title
+|EVES Purpose and Process
+
+|Type
+|Process
+
+|Linked to
+|-
+
+|Status
+|Draft
```

Comment:
Give hints about the expected input

Reply:
Or do you mean as a sort of comment? I have to check if this is possible in AsciiDoc.